### ISSUE TITLE
Odd custom module behavior

### ISSUE THREAD

**Issue by JAyenGreen** [backdrop/backdrop-issues#2368]
I'm certain I've munged something, in ignorance, but so far cannot determine what. My implementation is running fine on Pantheon. I push to it with git. I downloaded an export of the database from git, imported it locally, and added a local.settings.php file pointing to it.

What's happening is that I'm getting an error with my custom module that a call from the module to one of the inc files is to a nonexistent function, which it's not, and despite the module_load_include call prior to the function call. Running drush results in the same. If I print BACKDROP_ROOT it's correct. If I print backdrop_get_path('module', 'my_module') it is correct. However, if I add 

     $ret =
 
before the module_load_include, $ret is empty, and should, I believe, contain the file name.

**Comment by JAyenGreen** [comment]
Oh. Left out what is probably the important part. Backdrop first complained about the absence locally of the config directory from the Pantheon implementation. I'm guessing it and the rest of the files directory is ignored. I copied it down, and Backdrop seemed happy. I did note the comment about deploying to staging when deploying that directory, but I wasn't certain whether I should do that given I was importing the db?
